src/tests/Jenkinsfile

```diff
@@ -1,33 +1,64 @@
+properties([
+  pipelineTriggers([[$class: 'PeriodicFolderTrigger', interval: '1d']]),
+  parameters([
+    string(name: 'API_URL', defaultValue: 'https://api.quantum-inspire.com/', description: 'Quantum Inspire API url'),
+    string(name: 'API_CREDENTIALS_ID', defaultValue: 'api.quantum-inspire.com', description: 'Jenkins credentials ID'),
+  ])
+])
 pipeline {
     agent {
         dockerfile {
               filename 'src/tests/Dockerfile.build'
               args '-v pip_cache:/var/pip_cache'
         }
     }
+    environment {
+       PERSIST_WORKSPACE = ""
+    }
     stages {
-        stage('Build') {
+        stage('Checkout modules') {
             steps {
                 sh 'virtualenv venv'
-                sh '. ./venv/bin/activate && pip install .[qiskit,projectq] --cache-dir /var/pip_cache'
-                sh 'pwd'
-                sh 'ls -lah'
-            }
-        }
-        stage('Tests') {
-            steps {
                 sh '''
-                       . ./venv/bin/activate &&
-                       coverage run --source="./src/quantuminspire" -m unittest discover -s src/tests -t src -v &&
-                       coverage report -m &&
-                       coverage xml
+                    . ./venv/bin/activate &&
+                    pwd &&
+                    ls -lah &&
+                    pip install qiskit &&
```

Comment:
Why install qiskit and then uninstall it again?

If there is a valid reason for doing this, it would be good to add a comment I think, because it is unexpected.

Regardless, should probably include `--cache-dir /var/pip_cache` and possibly a `--upgrade` here to avoid all the `The directory '/.cache/pip/http' or its parent directory is not owned by the current user and the cache has been disabled. Please check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.` errors?

Reply:
oke, i'll add cache dir and upgrade

Reply:
Added comment:
# Installing qiskit, qiskit-aer and qiskit-terra as separated (development) packages doesn't work as expected.
# As a workaround we install the qiskit packages from PyPI, uninstall package qiskit and reinstall
# the development version of qiskit afterwards.